src/app/public/src/modules/sidebar/sidebar-wrapper.component.scss

```diff
@@ -1,87 +1,45 @@
 @import '../../styles/_variables';
 
 .stache-sidebar-wrapper {
-  position: fixed;
+  background-color: #fff;
+  position: absolute !important;
   z-index: $stache-sidebar-z-index;
-  bottom: 0;
   left: 0;
+  top: 0;
+  bottom: 0;
   border-right: 1px solid $stache-border-color-neutral-medium;
```

Comment:
Did Todd approve a drop shadow on the sidebar? I'm having trouble remembering...

Reply:
He did for the last iteration, though with it no longer overlapping the sidebar or top stuff beyond the layout I was experimenting with removing it.

Reply:
If Todd's okay with that...

Reply:
Todd asked for it when I was showing him sidebar last time, so I added it back.